testdata/postgres-docker-entrypoint.d/enable_pg_stat_statements.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+echo "shared_preload_libraries = 'pg_stat_statements'" >> $PGDATA/postgresql.conf
+echo "pg_stat_statements.max = 10000" >> $PGDATA/postgresql.conf
+echo "pg_stat_statements.track = all" >> $PGDATA/postgresql.conf
+echo "pg_stat_statements plugin is enabled, please call this query in postgres \"CREATE EXTENSION pg_stat_statements SCHEMA public;\""
```

Comment:
Please always add \n to the last lines of files. Without them, many UNIX command-line tools work incorrectly.